src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.Extensions.Configuration.AzureAppConfiguration.Models;
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.Tracing;
```

Comment:
I don't see any other changes related to this namespace in this file. Is it needed?